src/fu-tool.c

```diff
@@ -1515,10 +1396,6 @@ main (int argc, char *argv[])
 	g_signal_connect (priv->cancellable, "cancelled",
 			  G_CALLBACK (fu_util_cancelled_cb), priv);
 
-	/* sort by command name */
-	g_ptr_array_sort (priv->cmd_array,
-			  (GCompareFunc) fu_sort_command_name_cb);
-
```

Comment:
Why did you move the sorting into when the command was added?  It seems that will cause it to be called a lot more than is necessary.

Reply:
Good question. I made it automatic as the overhead is literally not measurable, and it's one less thing for the CLI to forget to do. If you think it's better explicit (e.g. a CLI might not *want* the commands sorted), I can move it out again.

Reply:
Are you thinking we might have a 3rd tool some day and don't want to forget it there?  It seems like good coding practice to not call things that run every time the app is opened more than is needed even if it's not a measurable impact.

Reply:
Fair enough, I'll move it out and be explicit.

Reply:
In fact, after doing this; do we really want the commands to be sorted? If we leave them alone, we can group them in the sections of the things they're doing, e.g. `enable-remote` gets put before `disable-remote`.

Reply:
That's a great point.  I do think they're better grouped on what they do.

Reply:
Okay, I've put the sorting back in, and in a different PR we can discuss the grouping, and disable the sorting.